<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profile</class>
 <widget class="QDialog" name="Profile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>400</width>
       <height>400</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">
	background-color: rgb(35, 35, 35);
	border-radius:30px;
background-color: rgb(45, 45, 45);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>400</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame{
	background-color: rgb(35, 35, 35);
	border-radius:5px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <widget class="QLabel" name="profile">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>50</y>
              <width>231</width>
              <height>221</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border-radius: 10px;
	padding-left:10px;
}</string>
            </property>
            <property name="text">
             <string>Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_close">
            <property name="geometry">
             <rect>
              <x>450</x>
              <y>10</y>
              <width>17</width>
              <height>17</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(255, 0, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 0, 0,150);
	
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_maximize">
            <property name="geometry">
             <rect>
              <x>427</x>
              <y>10</y>
              <width>17</width>
              <height>17</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	
	background-color: rgb(255, 170, 0);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(255, 170, 0,150);
	
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QPushButton" name="btn_minimize">
            <property name="geometry">
             <rect>
              <x>404</x>
              <y>10</y>
              <width>17</width>
              <height>17</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>17</width>
              <height>17</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	background-color: rgb(85, 255, 127);
	border:none;
	border-radius: 8px;
}

QPushButton:hover{	
	background-color: rgba(85, 255, 127,150);
	
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>0</y>
              <width>40</width>
              <height>40</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/user-check.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="middlename">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>110</y>
              <width>191</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Middlename</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="contact">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>230</y>
              <width>191</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Contact</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="firstname">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>50</y>
              <width>191</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Firstname</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="awards">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>350</y>
              <width>81</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Awards</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>350</y>
              <width>40</width>
              <height>40</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/award.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>170</x>
              <y>350</y>
              <width>40</width>
              <height>40</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/tag.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="role">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>350</y>
              <width>91</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Role</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="lastname">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>170</y>
              <width>191</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Lastname</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="email">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>290</y>
              <width>281</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>example@example.com</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="visits_count">
            <property name="geometry">
             <rect>
              <x>370</x>
              <y>350</y>
              <width>91</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;

}</string>
            </property>
            <property name="text">
             <string>Visits</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>350</y>
              <width>40</width>
              <height>40</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resource/asset.qrc">:/icons/asset/activity.svg</pixmap>
            </property>
           </widget>
           <widget class="QLabel" name="last_seen">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>410</y>
              <width>311</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Last seen</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="duration">
            <property name="geometry">
             <rect>
              <x>340</x>
              <y>410</y>
              <width>121</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
}</string>
            </property>
            <property name="text">
             <string>Duration</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="visits">
            <property name="geometry">
             <rect>
              <x>320</x>
              <y>290</y>
              <width>141</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Visits</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="program">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>470</y>
              <width>441</width>
              <height>41</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(45, 45, 45);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Department</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="firstname_2">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>0</y>
              <width>191</width>
              <height>31</height>
             </rect>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
	color: rgb(255, 255, 255);
	background-color: rgb(35, 35, 35);
	border-radius: 10px;
	padding-left:10px;

}</string>
            </property>
            <property name="text">
             <string>Profile</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/asset.qrc"/>
 </resources>
 <connections/>
</ui>
